src/taco-cli/cli/utils/cordovaWrapper.ts

```diff
@@ -34,18 +34,18 @@ import ConfigParser = Cordova.cordova_lib.configparser;
 import packageLoader = tacoUtility.TacoPackageLoader;
 
 class CordovaWrapper {
-    private static CordovaCommandName: string = os.platform() === "win32" ? "cordova.cmd" : "cordova";
-    private static CordovaRequirementsMinVersion: string = "5.1.1";
-    private static CordovaNpmPackageName: string = "cordova";
+    private static cordovaCommandName: string = os.platform() === "win32" ? "cordova.cmd" : "cordova";
```

Comment:
We talked offline, members that are more of an "result" rather than a pure constant should be named like regular variables